parsers/powershell.c

```diff
@@ -236,6 +241,26 @@ static void parseString (vString *const string, const int delimiter)
 	}
 }
 
+/* parse a identifier that may contain scopes, such as function names and
+ * variable names.
+ *
+ * 	VariableName
+ * 	FunctionName
+ * 	local:VariableName
+ * 	private:FunctionName
+ */
+static void parseScopeIdentifier (vString *const string, const int firstChar)
+{
+	int c = firstChar;
+	do
+	{
+		vStringPut (string, (char) c);
```

Comment:
The cast operation, (char), is not needed. vStringPut takes an integer as the second arg.

Reply:
I think a commit should be perfect and should not depend on a successor commit (if possible).

[283e2fd](https://github.com/universal-ctags/ctags/pull/3531/commits/283e2fd0710119bd5e0f38f91e80e695340f4570) is not perfect; it includes an unnecessary cast operation. It is fixed in the successor commit ([6c60f00](https://github.com/universal-ctags/ctags/pull/3531/commits/6c60f004c9ed79b148ceca8a3212fd756cc76a90)). 

If a new potential contributor reads [283e2fd](https://github.com/universal-ctags/ctags/pull/3531/commits/283e2fd0710119bd5e0f38f91e80e695340f4570) to study how to improve the Powershell parser, the one may think (char) should be added to the second arg of vStringPut.

I think you should reorder the commits in this pull request.

Make a commit [6c60f00](https://github.com/universal-ctags/ctags/pull/3531/commits/6c60f004c9ed79b148ceca8a3212fd756cc76a90) as a preparation change, then add the main commit to this pull request.

If you don't know how to do this, it is time to learn "git rebase -i ...".
I think it is the most brilliant feature git provides.